float64[] joint_values # ordered joints of the manipulator
float64[] base_pos     # [x,y,z] base position
